requirements.txt

```diff
@@ -14,7 +14,6 @@ pandas
 psutil
 PySide2>=5.13.2,<=5.14.1; platform_machine != 'arm64'
 PySide2>=5.13.2,<=5.15.5; sys_platform == 'darwin' and platform_machine == 'arm64'
-qtpy>=2.0.1
```

Comment:
why did we remove qtpy from requirements.txt?

Reply:
we found a [qtpy conda package](https://anaconda.org/anaconda/qtpy) (>=v2.0.1) that has support for both windows, ubuntu, and mac, so we moved this requirement to the conda environment yamls